Plugin/src/main/java/com/gmail/filoghost/holographicdisplays/bridge/protocollib/current/ProtocolLibHookImpl.java

```diff
@@ -57,8 +59,18 @@
 	private NMSManager nmsManager;
 	private PacketHelper packetHelper;
 	private MetadataHelper metadataHelper;
-	
-	
+	private final boolean useOptional;
+
+	public ProtocolLibHookImpl() {
+		String version = Bukkit.getServer().getClass().getPackage().getName().replace(".", ",").split(",")[3];

```

Comment:
Why first replace `.` with `,` and then split on it?
Could you not directly split at `.` by using `\\.`?

Reply:
Honestly just reused code from here: https://github.com/Niall7459/HolographicExtension/blob/f25850f25ec91ac2dee9bee5f4314312713eb0f3/src/main/java/net/kitesoftware/holograms/listener/PacketPlaceholderListener.java#L40

(which is also GPL 3)